LTX/data/LTX/terrain.ruleset

```diff
@@ -884,10 +1083,109 @@ Tundra are broad, cold regions, fit for some agriculture and little\
  else.\
 ")
 
-; /* <-- avoid gettext warnings
-;
-; Resource types:
-;
+[terrain_taiga]
+name                 = _("Taiga")
+graphic              = "taiga"
+graphic_alt          = "forest"
+identifier           = "y"
+class                = "Land"
+movement_cost        = 4
+defense_bonus        = 25
+food                 = 1
+shield               = 2
+trade                = 0
+resources            = "Game", "Furs", "Forest Game", "Ivory", "Coal", "Oil", "Iron", "Gems", "Peat"
+road_food_incr_pct   = 0
+road_shield_incr_pct = 0
+road_trade_incr_pct  = 1
+base_time            = 2
+road_time            = 6
+cultivate_time       = 10
+plant_time           = 0
+irrigation_result    = "Tundra"
+irrigation_food_incr = 0
+irrigation_time      = 10
+mining_result        = "Swamp"
+mining_shield_incr   = 0
+mining_time          = 25
+transform_result     = "Grassland"
+transform_time       = 24
+pillage_time         = 1
+clean_pollution_time = 3
+clean_fallout_time   = 3
+animal               = "None"
+warmer_wetter_result = "Swamp"
+warmer_drier_result  = "Forest"
+cooler_wetter_result = "Tundra"
+cooler_drier_result  = "Tundra"
+native_to            = "Land", "Small Land", "Merchant", "Big Land", "Air", "Missile", "Helicopter", "Big Siege", "Small Unit", "Amphibious", "Nuclear", "Ancient Land"
+flags                = "Starter", "CanHaveRiver"
+property_cold        = 30
+property_wet         = 15
+property_foliage     = 15
+property_temperate   = 10
+color.r              = 43
+color.g              = 107
+color.b              = 19
+helptext             = _("\
+Taiga forests are swampy coniferous forest of high northern latitudes, making agriculture \
+ problematic.\
+")
+
+[terrain_volcano]
+name                 = _("Volcano")
+graphic              = "volcano"
+graphic_alt          = "mountains"
+identifier           = "v"
+class                = "Land"
+movement_cost        = 6
+defense_bonus        = 100
+food                 = 0
+shield               = 0
+trade                = 0
+resources            = "Gold", "Gold","Gold", "Gold", "Iron", "Iron", "Iron", "Iron", "Copper", "Uranium", "Gems"
```

Comment:
So what does having multiples of the same resource do?

Reply:
Change their generation probability.